assets/csidriveroperators/openstack-cinder/07_deployment.yaml

```diff
@@ -39,6 +39,12 @@ spec:
               fieldPath: metadata.name
         image: ${OPERATOR_IMAGE}
         imagePullPolicy: IfNotPresent
+        volumeMounts:
+        - name: secret-cinderplugin
+          mountPath: /etc/openstack
+          readOnly: true
+        - name: cacert
+          mountPath: /etc/kubernetes/static-pod-resources/configmaps/cloud-config
```

Comment:
Is there an accompanying PR in the operator that uses these files?

Reply:
Here is is: https://github.com/openshift/openstack-cinder-csi-driver-operator/pull/10